packages/atomic/src/components/result-template-components/atomic-result-badge/atomic-result-badge.tsx

```diff
@@ -47,16 +48,18 @@ export class AtomicResultBadge {
     );
   }
 
+  private getTextContent() {
+    if (this.field !== undefined) {
+      return <atomic-result-text field={this.field}></atomic-result-text>;
+    }
+    if (this.label !== undefined) {
```

Comment:
Alternatively, we could specify `label` or the slot as a fallback for when the field is missing from a result.

Reply:
I think it's better how it is right now versus too much hidden logic about what gets displayed.